lms/static/scripts/frontend_apps/components/LMSGrader.js

```diff
@@ -17,11 +17,31 @@ export default function LMSGrader({
   children,
   assignmentName,
   courseName,
-  students,
+  students: unorderedStudents,
 }) {
   // No initial current student selected
   const [currentStudentIndex, setCurrentStudentIndex] = useState(-1);
 
+  // A sorted list of students. Students are sorted by
+  // displayName.
+  const [students] = useState(() => {
+    // Make a copy
+    const students_ = [...unorderedStudents];
+    students_.sort((student1, student2) => {
+      function compareNames(name1, name2) {
+        if (name1.toLowerCase() < name2.toLowerCase()) {
+          return -1;
+        } else if (name1.toLowerCase() > name2.toLowerCase()) {
+          return 1;
+        } else {
+          return 0;
+        }
+      }
+      return compareNames(student1.displayName, student2.displayName);
```

Comment:
Are we 100% sure we always have a `displayName`? Maybe we are!

Reply:
Not likley, but its easy to make at least this code 
1. Not break
2. Sort it first

It would likely be a bad state for the app, but we can also attempt to make it less bad by not throwing confusing JS errors.